placideirandora/epic-mail
# #164774477 fixed the eslint issues

#### What does this PR do?
It fixes **ESLint** issues.

#### Description of Task to be done?
1. It turns on the **ESLint** rules specified by **Airbnb**.
2. It reflects on the **ESLint** rules update and refactors the code to comply with the **Airbnb** linting style.

#### How should this be manually tested?
1. Clone the repo.
2. Follow the instructions on how to set up the repo in the ReadMe.
3. Check to see if violates any **ESLint** rules such as **prefer-template**, and **line-break**.

#### What is the pivotal tracker story?
[Fixing the ESLint issues](https://www.pivotaltracker.com/n/projects/2314363/stories/164774477)
